filelist.txt

```diff
@@ -48,6 +48,8 @@ file /etc/gshadow etc/gshadow 644 0 0
 file /etc/fstab etc/fstab 644 0 0
 file /etc/hosts etc/hosts 644 0 0
 slink /etc/mtab /proc/mounts 755 0 0
+dir /etc/udhcp 755 0 0
+file /etc/udhcp/simple.script stage/etc/udhcp/simple.script 755 0 0
```

Comment:
Another example https://github.com/linaro-swg/gen_rootfs/blob/master/filelist.txt#L2
`file /bin/busybox stage/bin/busybox 755 0 0`
All stuff, that were created during build is put in `stage`(`stage/etc` and etc.) dir. `etc` dir in the root just contains static stuff